src/source/reference/put-campaign-targeting.rst

```diff
@@ -0,0 +1,28 @@
+.. index:: PUT /campaign/{id}/targeting
+
+PUT /campaign/{id}/targeting
+===============================
+
```

Comment:
include / refer to it being experimental still

Reply:
unexposed (but added warning)